en/docs/get-started/quick-start-guide.md

```diff
@@ -188,7 +199,7 @@ First you need to open the `<MI_QSG_HOME>/HealthcareIntegrationProject` folder i
         micro-integrator.bat
         ```
 
-### Step 3 - Observe deployed artifacts (Optional)
+## Step 3 - Observe deployed artifacts (Optional)
 
 You can install and start the Integration Control Plane (ICP) to observe details of the deployed artifacts by following the steps below:
 
```

Comment:
Once you have deployed the artifacts and started the Micro Integrator server, you need to stop it before starting the ICP server.
->
After deploying the artifacts and starting the Micro Integrator server, you need to stop it before starting the ICP server.